pkg/managed/namespace_test.go

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package managed
+
+import (
+  "context"
+  "github.com/stretchr/testify/assert"
+  "github.com/verrazzano/verrazzano-operator/pkg/types"
+  "github.com/verrazzano/verrazzano-operator/pkg/util"
+  v1 "k8s.io/api/core/v1"
+  metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+  "testing"
+)
+
+func TestCreateNamespaces(t *testing.T) {
+  type args struct {
+    mbPair              *types.ModelBindingPair
+    filteredConnections map[string]*util.ManagedClusterConnection
+  }
+  tests := []struct {
+    name    string
+    args    args
+    wantErr bool
+  }{
+    {
+      name: "create namespaces",
+      args: args{
+        mbPair:              getModelBindingPair(),
+        filteredConnections: getManagedClusterConnections(),
+      },
+      wantErr: false,
+    },
+  }
+  for _, tt := range tests {
+    t.Run(tt.name, func(t *testing.T) {
+      if err := CreateNamespaces(tt.args.mbPair, tt.args.filteredConnections); (err != nil) != tt.wantErr {
```

Comment:
i think (err != nil) && t.wanterr is clearer

Reply:
That changes the logic.  If there is no error but I want one then I want the test to fail.